flycheck.el

```diff
@@ -4408,6 +4459,13 @@ default `:verify' function of command checkers."
                                (and verify-fn
                                     (funcall verify-fn checker)))))))
 
+  (let ((enabled-fn (plist-get properties :enabled)))
+    (setq properties
+          (plist-put properties :enabled
+                     (lambda (checker)
+                       (and enabled-fn
+                            (funcall enabled-fn checker))))))
```

Comment:
I don't think that this is needed is it? After all, we're not adding to `:enabled` here, do we?